Nodejs/Product/Nodejs/TypingsAcquisition.cs

```diff
@@ -14,34 +14,39 @@
 //
 //*********************************************************//
 
-using System.Threading.Tasks;
-using Microsoft.VisualStudioTools.Project;
-using Microsoft.NodejsTools.Npm;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.IO;
-
-using SR = Microsoft.NodejsTools.Project.SR;
-using System;
 using System.Security;
 using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.VisualStudioTools.Project;
+using Microsoft.NodejsTools.Npm;
+using Microsoft.NodejsTools.Npm.SPI;
+
```

Comment:
nit: can you remove the spaces between usings - I don't think VS's automatic-usings-sort works otherwise.